pom.xml

```diff
@@ -3,7 +3,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>1.526</version>
+        <version>1.598</version>
```

Comment:
I'm not sure we want plugin to depend on such a recent release. Won't be available on LTS for sample